src/app/(sidebar)/xdr/view/page.tsx

```diff
@@ -157,6 +179,42 @@ export default function ViewXdr() {
     return null;
   };
 
+  const renderJsonContent = ({
+    jsonArray,
+    xdr,
+  }: {
+    jsonArray: AnyObject[];
+    xdr: string;
+  }) => {
+    if (jsonArray.length > 1) {
+      return (
+        <div className="PrettyJson PrettyJson__array">
+          <div className="PrettyJson__inline">
+            <span className="PrettyJson__bracket">[</span>
+            <span className="PrettyJson__expandSize">{`${jsonArray.length} items`}</span>
+          </div>
+          {jsonArray.map((j, index) => (
+            <>
+              <PrettyJsonTransaction
+                // Using index here because we can't get something unique from the JSON
+                key={`pretty-json-${index}`}
+                json={j}
+                xdr={xdr}
+              />
+            </>
+          ))}
+          <span className="PrettyJson__bracket">]</span>
+        </div>
```

Comment:
Handling the array styling manually here because it's the only place where we have it. If we need it someplace else, we'll create a component for it.